index.js

```diff
@@ -0,0 +1,306 @@
+(function(f) {
+
+  'use strict';
+
+  /* istanbul ignore else */
+  if (typeof module === 'object' && typeof module.exports === 'object') {
+    module.exports = f();
+  } else if (typeof define === 'function' && define.amd != null) {
+    define([], f);
+  } else {
+    self.sanctuaryTypeClasses = f();
+  }
+
+}(function() {
+
+  'use strict';
+
+  var STC = {};
+  var instances = STC.instances = {Array: {}, Function: {}};
+
+  var FL = {
+    ap: 'ap',
+    chain: 'chain',
+    concat: 'concat',
+    empty: 'empty',
+    equals: 'equals',
+    extend: 'extend',
+    extract: 'extract',
+    map: 'map',
+    of: 'of',
+    reduce: 'reduce',
+    sequence: 'sequence'
+  };
```

Comment:
I've never used AMD, but the bundles produced by wzrd, for example, are AMD compatible